app/response_processor.py

```diff
@@ -102,6 +108,33 @@ def _encode(self, decrypted):
             raise QuarantinableError
         return xml
 
+    def _send_rm_receipt(self, case_id):
+        request_url = settings.RM_SDX_GATEWAY_URL
+
+        try:
+            r = self.session.post(request_url, auth=settings.BASIC_AUTH, json={'caseId': case_id})
+        except MaxRetryError:
+            logger.error("Max retries exceeded (5)",
+                         request_url=request_url,
+                         case_id=case_id)
+            raise RetryableError
+
+        if r.status_code == 200 or r.status_code == 201:
+            logger.info("RM sdx gateway receipt creation was a success",
+                        request_url=request_url,
+                        case_id=case_id)
+            return
+
+        elif 400 <= r.status_code < 500:
+            logger.error("RM sdx gateway returned client error, unable to receipt",
+                         request_url=request_url,
+                         status=r.status_code,
+                         case_id=case_id)
+            raise QuarantinableError
+        else:
+            logger.error("Service error", request_url=request_url, case_id=case_id)
```

Comment:
I think a better log message here would be useful, something like:

"rm sdx receipting service error - retrying"

Reply:
This is expected by splunk to generate ops channel errors